config/env/default.js

```diff
@@ -28,5 +28,19 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUpload: {
+      dest: './modules/users/client/img/profile/uploads/', //Profile upload destination path
+      limits: {
+        fileSize: 1*1024*1024 //Max file size in bytes
+      },
+      fileFilter: function (req, file, cb) {
+        if (file.mimetype !== 'image/png' && file.mimetype !== 'image/jpg' && file.mimetype !== 'image/jpeg' && file.mimetype !== 'image/gif') {
+            return cb(new Error('Only image files are allowed!'), false);
+        }
+        cb(null, true);
+      }
+    }
```

Comment:
@gustavodemari I like the 2nd option. 

What do you think about adding a env config setting for a list of accepted file types for `profileUpload`? In the Multer configuration file, you can retrieve the list of accepted file types from the env config. 

Also, I think using lodash in your `fileFilter` function to check the current file's *mimetype* against the list. WDYT?